static/app/views/performance/landing/widgets/widgets/vitalWidget.tsx

```diff
@@ -313,11 +308,13 @@ export function VitalWidget(props: PerformanceWidgetProps) {
   );
 }
 
-function getVitalDataForListItem(listItem: TableDataRow) {
+function getVitalDataForListItem(listItem: TableDataRow, vital: WebVital) {
   const poorData: number =
-    (listItem.count_if_measurements_lcp_greaterOrEquals_4000 as number) || 0;
-  const mehData: number = (listItem['equation[0]'] as number) || 0;
-  const goodData: number = (listItem['equation[1]'] as number) || 0;
+    (listItem[`count_web_vitals_${vital.replace('.', '_')}_poor`] as number) || 0;
```

Comment:
Can we use `getAggregateAlias` from discover/fields.tsx here instead?

Reply:
yeh